### DISCUSSION TITLE
Grazing in landuse timeseries files

### DISCUSSION THREAD

**Discussion by olyson** [ESCOMP/CTSM#1950] [Q&A] [answered]
Are there any future plans to use the currently unused GRAZING field in the landuse timeseries files (user question).

**Comment by ekluzek** [comment, marked as answer]
This is a future desire. But requires development. I don't think it comes at all with CTSM5.2 but hopefully in the next generation of datasets.

@lawrencepj1 can you comment on this?

**Reply by lawrencepj1** [reply]
Hi @ekluzek

Yes exactly right. This will be an important effort for post CTSM5.2.

Peter

**Comment by olyson** [comment]
Thanks all!
